### DISCUSSION TITLE
Shift click on tabs

### DISCUSSION THREAD

**Discussion by jcrowson12MSU** [anvilistas/anvil-extras#499] [Ideas]
Is there a reason that tabs do not recognize shift or control clicks? Is there a quick work around for this?

**Comment by jcrowson12MSU** [comment]
I figured it out by overriding the _raise_tab_click method and including the keys.

**Comment by s-cork** [comment]
Did you want to PR the fix?

**Reply by jcrowson12MSU** [reply]
Maybe. I need to figure out how.

**Reply by s-cork** [reply]
Let us know if you need help. There should be a contributing guide in the docs.
